GUIDE.md

```diff
@@ -19,7 +19,7 @@ Procedure
 
 As `$USER` user with `sudo` privileges,
 
-1.  Generate an SSH key if not already available (`ssh_key/key`)
+1.  Generate an SSH key if not already available (`~/.ssh/id_rsa`)
```

Comment:
it doesn't exits on fresh beaker as if i remember right.

Reply:
ack
creating it on ``install_environment``